content/en/blog/_posts/2022-01-24-POV-Kubernetes-cluster-infrastructure-upgrade-strategies.md

```diff
@@ -0,0 +1,94 @@
+## Context
+
+This blog is an outcome of the experience gained while working for a client having a Kubernetes based platform hosted on infrastructure (EC2 instances) hosted on AWS. AWS releases new images to be used to build the servers (EC2 instances) every now and then, when a new vulnerability fix is made, a newer feature is added etc. This client had additional hardening to be done on those images and a custom image is released internally for the entire organization to use. There was a deadline imposed by when old the images must be stopped to be in use and the infrastructure is to be built using only the latest image available. One of the teams that is impacted by this was the platform team, the team that hosts and manages a shared platform based on Kubernetes. Many applications teams in the organization host their applications on this shared platform as independent containers. 
```

Comment:
“images” here could be confusing. What kind of image do you mean?

Reply:
AWS releases new images to be used to build the servers (EC2 instances) every now and then, when a new vulnerability fix is made, a newer feature is added etc 
+1 to what @sftim said. Once we clarify on what kind of image is implied, we also need to rephrase this sentence because it seems convoluted as of now.